### ISSUE TITLE
Search bar appearance

### ISSUE THREAD

**Issue by yosl** [junior-dev-struggle-bus/junior-dev-struggle-blog#67]
The search bar is kinda jank

let's make it, say, a search icon that expands into a search bar

bonus points for no javascript

**Comment by Xoadra** [comment]
I was thinking about going in and improving the search bar's appearance, but my plan was just to make it larger, more crisp looking, and scale down for certain screen sizes.

The icon and animation could always be added later, but if you'd rather the icon and animation go with the first update to it, I can hold off on it until I have an idea of how I want to do it or to allow someone else to tackle the matter.

**Comment by AvianAnalyst** [comment]
I don't think it should be limited to just icon/animation until someone's on a phone, or at minimum a tablet. If you're on a desktop/laptop that introduced a lot of dead space which I think feels less balanced.
